authz/audit/stdout/stdout_logger_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+ *
+ * Copyright 2023 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package stdout
+
+import (
+	"bytes"
+	"encoding/json"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/grpc/authz/audit"
+	"google.golang.org/grpc/internal/grpctest"
+)
+
+var (
+	content     = json.RawMessage(`{"name": "conf", "val": "to be ignored"}`)
+	builder     = &loggerBuilder{}
+	config, _   = builder.ParseLoggerConfig(content)
+	auditLogger = builder.Build(config)
+)
```

Comment:
Please try and move as much of these as possible into individual tests. It is better for tests to not share common dynamic state. Its OK for them to share static state like configuration etc.